hilti/toolchain/include/ast/node-tag.h

```diff
@@ -126,7 +126,6 @@ constexpr Tag BuiltInFunction = 401;
 constexpr Tag Coerced = 402;
 constexpr Tag ConditionTest = 403;
 constexpr Tag Ctor = 404;
-constexpr Tag Deferred = 405;
```

Comment:
Nit: How do you feel about decrementing all following tags in this group to avoid the hole?

Reply:
Will do, I'm fine either way.